selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/parameters/RenameParametersTest.java

```diff
@@ -199,9 +199,12 @@ public void checkRenameParameters15() throws Exception {
     events.clearAllMessages();
     editor.waitActive();
     editor.goToCursorPositionVisible(15, 23);
-    editor.launchLocalRefactor();
-    editor.typeTextIntoEditor("i");
-    editor.typeTextIntoEditor(Keys.ENTER.toString());
+    editor.launchRefactorForm();
```

Comment:
Could you, please, divide separate steps by empty rows and add short comment on the top of steps with explanation what the steps are aimed to do?

Reply:
OK, I'll correct